server.js

```diff
@@ -74,17 +74,48 @@ const getIndexBody = function(variables) {
   return template;
 };
 
+const getRates = () => {
+  return new Promise((resolve, reject) => {
+    http.get('http://www.cnb.cz/cs/financni_trhy/devizovy_trh/kurzy_devizoveho_trhu/denni_kurz.txt', (response) => {
```

Comment:
The response does not contain CORS headers => I cannot fetch this directly from the browser.